src/components/Forms/SuggestionBox.js

```diff
@@ -0,0 +1,192 @@
+      <input type="hidden" name="bot-field" />
+      <input type="hidden" name="form-name" value="contact" />
+      {/* netlify form handling */}
+
+      <Text fontSize="xl" textAlign="center">
+        Suggestion Box
+      </Text>
+
+      {/* renders when form is submitted with validation errors */}
+      {validationMessage && <Text textAlign="center">{validationMessage}</Text>}
+
+      <Flex direction="column" margin={theme.spacing.base}>
+        <FormLabel isRequired htmlFor="topic">
+          Topic
+        </FormLabel>
+        <Input
+          value={topic}
+          id="topic"
+          type="text"
+          placeholder="e.g. Business, Functionality, New Feature"
+          onChange={event => setTopic(event.currentTarget.value)}
+        />
+      </Flex>
+
+      <Flex direction="column" margin={theme.spacing.base}>
+        <FormLabel isRequired htmlFor="textField">
+          Description
+        </FormLabel>
+        <Textarea
+          value={description}
+          id="description"
+          placeholder="Description"
+          onChange={event => setDescription(event.currentTarget.value)}
+        />
+      </Flex>
+
+      <Flex direction="column" margin={theme.spacing.base}>
+        <FormLabel isRequired htmlFor="benifits">
+          Benifits
+        </FormLabel>
+        <Textarea
+          value={benifits}
+          id="benifits"
+          placeholder="Benifits"
+          onChange={event => setBenifits(event.currentTarget.value)}
+        />
+      </Flex>
+
+      <Flex direction="column" margin={theme.spacing.base}>
+        <FormLabel isRequired htmlFor="urgency">
+          Urgency: 1 (Most Urgent) - 5 (Not Urgent)
+        </FormLabel>
+        <NumberInput
+          id="urgency"
+          type="number"
+          min={1}
+          max={5}
+          value={urgency}
+          onChange={value => setUrgency(value > 5 ? 5 : value)}
+        >
+          <NumberInputField />
+          <NumberInputStepper>
+            <NumberIncrementStepper />
+            <NumberDecrementStepper />
+          </NumberInputStepper>
+        </NumberInput>
+      </Flex>
+
+      <Flex width="100%" direction="column">
+        <Flex direction="column" margin={theme.spacing.base}>
+          <FormLabel htmlFor="name">Name</FormLabel>
+          <Input
+            value={name}
+            id="name"
+            type="text"
+            placeholder="Name"
+            onChange={event => setName(event.currentTarget.value)}
+          />
+        </Flex>
+        <Flex direction="column" margin={theme.spacing.base}>
+          <FormLabel htmlFor="email">Email</FormLabel>
+          <Input
+            value={email}
+            id="email"
+            type="text"
+            placeholder="Email"
+            onChange={event => setEmail(event.currentTarget.value)}
+          />
+        </Flex>
+
+        <Flex width="100%" justify="center">
+          <PrimaryButton onClick={handleSubmit}>Submit</PrimaryButton>
```

Comment:
I see that you have the conditional set to provide the user with a confirmation message once the form is submitted, however upon testing the modal closed without any confirmation.